components/director/pkg/tenant/entity.go

```diff
@@ -28,6 +28,12 @@ const (
 	Account Type = "account"
 	// Subaccount tenants must have a parent of type Account.
 	Subaccount Type = "subaccount"
+	// Organization tenants can be parents of Folder tenants.
+	Organization Type = "organization"
+	// Folder tenants must have a parent of type Organization.
+	Folder Type = "folder"
+	// ResourceGroup tenants must have a parent of type Folder.
```

Comment:
// ResourceGroup tenants must have a parent of type Folder. -> // ResourceGroup tenants must have a parent of type Folder or organization.